### PR TITLE
Fixed some divisions and PlanningError str

### PR DESCRIPTION
**Pull request by cielavenir**
Those were newly found after fixing pickworker test on py3. On 2022 it was not found as sensors were not working on py3 at that time.

### CODE REVIEW

**Comment by cielavenir** [comment]
autotester

```
-----BEGIN PGP MESSAGE-----

hQIMAwAinXiLUbjzARAA0d/o5BNU1j68lYwGainGYq/Km65uZwuwRxVQuk3RkqYR
gfKMA4AZLqd1HZjD68rdv+A/XQwOOTqGfc3358vE8h2bkt0aWVagqAkgZKOdEEL2
DiFKY1zxMkTsIHcrwKZPzpyRPoMHwy2x4wyQHpS3gFyuoToSVLZ2CGro1uXZGvip
SOSAWMuOCDAwOGNDbeKrgwgzA0wMHIyFNNG7odeas1X+UF0oD0OszEF9Y+hSls8N
7u7Tur+RPZFXfyB6cTfz+pHEp7r4ubbyW/Yn+D69GduGFlw98jKtQ/TQXGy3pEN7
vktdFfVDscqTbFd9k1SvrxqVh2pbi7XBpr5o7mg40HCahvkdA10M7V+e0tM5TvRR
zgMb7iEGNRSAT57UsutZ8glq6psct1i23ZkRJn6dUdXSzBs7mbvLjAwy7tn2Fks+
KL/fPpwKc1znQSK/NhX/Z9NEXVIxHn037Gi8IAr6iuXXjkwhTIH0Xyo99yIG6tr5
zK97laQJLJgqe82g8PlL/4Cs5x3ZV9fvM0tMS4Y6DhmOIULCnj4y1fiJlj4xfGiT
uIcLmswYznc14CFcGnPOtbXKLZe1nKD6NUr8XR3pChnkjUIL1BU3OJGMnlHLFjiB
7MljnJ6AJn3sI5FNoGdUn/QkX/rmcc5B1UXi0Nic/xvAmLnXPIAeu1FWpT7u+eDS
ggHL0kFCSaTzqTvh01uBdGUe2kFMepitgq+L0GCxmjG2tlK9cich1GU46kN7hCwe
pgqQwbhgErWG8TSf1Ogmm58dwkptbh1SjkNhGFJMfOGJlb82u2PRNi5uHsfn8TAC
Tuiy1OwGvQfA+sPmJvhZRQaE1psNahzif7nhigb/uRW+2dU=
=cC3P
-----END PGP MESSAGE-----
```

File: python/openravepy.__init__.py [hunk 1]
```diff
@@ -142,7 +142,10 @@ def __unicode__(self):
         return s
         
     def __str__(self):
-        return unicode(self).encode('utf-8')
+        if sys.version_info[0]>=3:
+            return self.__unicode__()
+        else:
+            return unicode(self).encode('utf-8')
```
Comment (cielavenir) [inline python/openravepy.__init__.py:148]: this is kind of simulation of https://github.com/benjaminp/six/blob/1.16.0/six.py#L958 , we could use six instead

Comment (rdiankov) [reply python/openravepy.__init__.py:148]: sure

Comment (rdiankov) [reply python/openravepy.__init__.py:148]: @cielavenir feel free to make the change

Comment (cielavenir) [reply python/openravepy.__init__.py]: right I should have used six here, six is already used in some places..

**Comment by cielavenir** [comment]
```
-----BEGIN PGP MESSAGE-----

hQIMAwAinXiLUbjzAQ//cz5a9nslOxgpCUN/4z5QdhseNPsacBGsgh/8yirSrtDf
xfd30yHjo4BsvuRv3vPW6XqSfybI5o+XFmo+z2PRDHLJe+aSytgaCW9KcP4+V6WE
CJn9IsVGAFGeXlCyxRwqjI+a3MB9bsXF7WAAZfAhSdwqbqSIOB4nM6Kly9Z4EyHQ
V9v6szW5zJhP+v7TtFdb7uHHeL1TCYsLg5hIdK5llJ42aIPpasv0hla+3cetvxJP
neoOjxYJGACaWTv2Bg7g1iaZ0RwGxxtthLY+1YNG07FZnxiS7sv2rE94CQ4TwDc+
fYiQBcXQooR2fOZ+vtHfqw3QKvbW6bFjQ3J7eCNmIgQqQbruJJSPpsyAvy9MPQc9
I4J0aMgcuIG+O2TJX2sZfJmt0Hwtadc+tc/03aYaaEvFe2RIhJHNYsZ/G6DznbsX
NcWTyMQfScHIxQ5bNT9he/FzU8nZt/REZihuY/H5Wzwkr7da6+/gZcvSNBRxElM1
3O9GrfVV3FfUGJfK78hV4S3oTKS+SXAHSNX2ojYjUdaGOIcrGsfimrMDpuXw5BrG
0eaGkuyrHJbD932wsWKkU36R7pVUXOa3vhzgtB4Dg7XwlJW4s4PX+pCAyVNcnQe/
jeL3RFX97FuvK7STdgFSxngVd26W1Ngj2o7PTVAqmouHx1+k86IOGL3S7FSbQo/S
gQGIfEupgs2R5GH7CsMpNEJYib7ddCCekN6G9PG+16OvFYnUn3ML/IXTYI+4ixOf
2dnp5KI78ZXC0TCZu2JipKECdnFbknRlHp5IAzYlalDytnfmkpq5p9FuDmGkbets
4zwTe3vhbgtVprOtq6/FymGhfer9LwU7EYSarJSBCblP+A==
=atJ9
-----END PGP MESSAGE-----
```

**Comment by rdiankov** [comment]
thanks~

**Pull request merged**